cmd/kubeadm/app/cmd/token.go

```diff
@@ -221,7 +221,13 @@ func RunCreateToken(out io.Writer, client clientset.Interface, token string, tok
 		}
 	}
 
-	// TODO: Validate usages here so we don't allow something unsupported
+	// Validate usages
+	for _, usage := range usages {
+		if err := bootstrapapi.ValidateUsage(usage); err != nil {
+			return err
```

Comment:
This will error out when the first invalid usage is checked. I'd prefer that we give the user error message including all the invalid usages the config. @luxas WDYT?

Reply:
@xiangpengzhao good point!

Reply:
@xiangpengzhao `ValidateUsage`  already give all the invalid usages in the err info.

Reply:
valid

Reply:
@wackxu I don't think so. Please double check it. Maybe I'm wrong though.

Reply:
@wackxu, @xiangpengzhao wants that if you have usages `foo,bar`; there should be two errors returned. Now the for loop terminates directly for `foo` with an error

Reply:
@luxas points out what I meant exactly :)
I did so in the dup part of my that PR but I removed the code now, so that I can't paste the example code here :(

Reply:
Addressed